.github/workflows/tests.yaml

```diff
@@ -10,7 +10,7 @@ env:
     fail-fast: true
     PHPUNIT_FLAGS: "-v"
     SYMFONY_PHPUNIT_DIR: "$HOME/symfony-bridge/.phpunit"
-    SYMFONY_REQUIRE: ">=6.0"
+    SYMFONY_REQUIRE: ">=6.2"
```

Comment:
why having this env variable instead of letting the CI rely on the setting defined in the composer.json ?

Reply:
I removed it. Thanks.